style.css

```diff
@@ -109,19 +109,11 @@ a {
   padding-left: 0;
 }
 
-#projects {
-  display: grid;
-  grid-template-columns: 1fr 1fr;
-  grid-template-rows: 1fr 1fr 1fr 1fr 1fr;
-  box-sizing: border-box;
-  align-items: center;
-  padding: 142px 142px;
-}
-
 .card {
   display: flex;
-  flex-direction: column;
-  max-width: 330px;
+  flex-direction: row;
+  align-self: center;
+  max-width: 1155px;
   background: #fff;
   border: 2px solid #dfe1e6;
   border-radius: 16px;
```

Comment:
- [ ] Kindly make sure your cards are responsive and look similar to the ones provided in Figma.
Your mobile version :point_down: 
![image](https://user-images.githubusercontent.com/97100091/193204884-922a36f4-4253-4b0c-b768-f10368eb2764.png)
Figma design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/193204977-395f2b06-2083-41bd-9292-5e0b3cc5aec5.png)